### PR TITLE
OSM Translation default to .OSM

### PR DESCRIPTION
**Pull request by sisskind**

### CODE REVIEW

File: js/hoot/control/Export.js [hunk 1]
```diff
@@ -215,6 +215,8 @@ Hoot.control.export = function (context, sidebar) {
             if (exportType===''){exportType=d3.select('.fileExportFileType').attr('placeholder');}
             if (transType===''){transType=d3.select('.fileExportTranslation').attr('placeholder');}
 
+            if (transType==='OpenStreetMap.org (OSM)'){d3.select('#fileExportFileType').value('Open Street Map (OSM)');}
```
Comment (rmrice) [inline js/hoot/control/Export.js]: OpenStreetMap is one word.

**Comment by sisskind** [comment]
@brianhatchl @rmrice image carousel is the cucumber test failing.  Is that a known issue?  Same for #914

File: js/hoot/control/Export.js (hunk 1 above)
Comment (sisskind) [inline js/hoot/control/Export.js]: Will fix throughout.

**Comment by rmrice** [comment]
Not sure, it's been a minute since I've looked at hoot cuke tests.

**Comment by brianhatchl** [comment]
run :cucumber:

**Comment by sisskind** [comment]
@brianhatchl can still export to this...I think this is just to default to it for ease of use.

**Pull request merged**